terraform/lib/dependabot/terraform/update_checker.rb

```diff
@@ -198,6 +199,13 @@ def git_commit_checker
             version_class: Version
           )
       end
+
+      def eligible_sources_from(requirements)
+        requirements.
+          map { |r| r.fetch(:source) }.
+          find_all { |x| ELIGIBLE_SOURCE_TYPES.include?(x[:type].to_s) }.
```

Comment:
real nit, expected `select` here as it's what we use elsewhere in core

```suggestion
          select { |source| ELIGIBLE_SOURCE_TYPES.include?(source[:type].to_s) }.
```